app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -164,21 +174,36 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 
 				var coursesHtml = "";
 
-				if (scope.showStaffTable == true) {
-					var instructorTypeHeader = '<div class="instructor-type-header">Instructor TBD</div>';
+				if (scope.showInstructorUndeterminedTable == true) {
+					var Unassigned = {};
```

Comment:
Do not capitalize the first letter of a variable. We don't have a proper style guide but use the rest of the file as a guide.